class-gravity-flow.php

```diff
@@ -7639,6 +7641,26 @@ public function evaluate_conditional_logic( $logic, $form, $entry ) {
 			return $do_action;
 		}
 
+		/**
+		 * Determines if a multiuser field is being used for conditional routing and has an exact value match of user ID
+		 * 
+		 * @since 2.5
+		 *
+		 * @return bool
+		 */
+		public function filter_value_match_multiuser( $is_match, $field_value, $target_value, $operation, $source_field, $rule ) {
+			
+			if ( $source_field->type !== 'workflow_multi_user' ) {
```

Comment:
It should also return early when the current rule is based on an entry property or meta
```suggestion
			if ( ! $source_field || $source_field->type !== 'workflow_multi_user' ) {
```